tests/pipeline/test_engine.py

```diff
@@ -796,27 +784,30 @@ def test_drawdown(self):
         assert_frame_equal(expected, result)
 
 
-class ParameterizedFactorTestCase(TestCase):
+class ParameterizedFactorTestCase(WithTradingEnvironment, ZiplineTestCase):
+    sids = Int64Index([1, 2, 3])
+
+    @classmethod
+    def make_equity_info(cls):
+        cls.equity_info = ret = make_simple_equity_info(
```

Comment:
since we already have make_simple_equity_info, I am not sure how much more clear the tests will be if we lift all the arguments to the class level.